jobs/ci-kubernetes-node-kubelet-conformance.sh

```diff
@@ -19,4 +19,23 @@ set -o pipefail
 set -o xtrace
 
 # PROJECT="k8s-jkns-ci-node-e2e"
-test/e2e_node/jenkins/conformance/conformance-jenkins.sh test/e2e_node/jenkins/conformance/jenkins-conformance.properties
```

Comment:
Instead of messing with docker in docker, pass in the docker binary and socket as a volume. We do this in a few other places, too.